### PR TITLE
Refactoring and Javascript Unit tests for Get Help Now JS file

### PR DESCRIPTION
**Pull request by fatimarafiqui**
This includes the following:
-  Refactoring of get_help_now.js. to make it more modular.
-  Specs and test suites mocking methods through fixtures for get_help_now.js.
-  Block comments with each test suite name and description.

selection_006

### CODE REVIEW

**Review by Anjali05 — APPROVED** [review]

**Review by Anjali05 — APPROVED** [review]

File: js/get_help_now.js [hunk 1]
```diff
@@ -1,26 +1,42 @@
+/**
+ * showGetHelpActionSection() displays the page section
```
Comment (MehaKaushik) [inline js/get_help_now.js:2]: Mentioning showGetHelpActionSection() was not required.

File: js/get_help_now.js [hunk 2]
```diff
@@ -1,26 +1,42 @@
...
+		$('#content-container').addClass('hide');
+    	$('#content-container > div > div').addClass('hide');
+    }, 600);
+}
 $(document).ready(function() {
 	$('.contact-button').on('click', function() {
-		var thisId = $(this).attr('id'),
-			getHelpSection = $('#get-help-section'),
-			getHelpActionSection = $('#get-help-action-section');
+		var thisId = $(this).attr('id');
```
Comment (MehaKaushik) [inline js/get_help_now.js:29]: Instead of using "this"Id, why not use "button"Id or "submitButton"id

File: tests/jsTests/getHelpNowSpec.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+/**
+ * Suite : Tests for show get help action section
+ * Description : This is a test suite for checking if the get-help-action section (call/sms) is visible
+ */
+describe("Tests for show get help action section", function() {
+	beforeEach(function() {
+		jasmine.clock().install();
+		setFixtures('<div id="get-help-section"></div> \
+				<div id="get-help-action-section"></div>');
+		spyOn($.fn, "fadeOut").and.callFake(function() {
```
Comment (MehaKaushik) [inline tests/jsTests/getHelpNowSpec.js:10]: Why are you calling the functions hide and show ? Why are you not calling on-click for contact button ?

**Review by MehaKaushik — CHANGES_REQUESTED** [review]

File: js/get_help_now.js (hunk 1 above)
Comment (fatimarafiqui) [reply js/get_help_now.js:2]: As discussed in earlier Pull requests and on slack, we are following AirBnB's style guide for comments. Like php comments, we are mentioning function's name here as well. Reference: [Airbnb style guide](https://github.com/airbnb/javascript#comments)

File: js/get_help_now.js (hunk 2 above)
Comment (fatimarafiqui) [reply js/get_help_now.js:29]: This is a widely used term in JS and Jquery. Just following the coding conventions and naming style, specifying that we are referencing the ID of 'this' particular element.

Also, this is not a submit button and there were many buttons. Using buttonId would have made it more ambiguous. We are just being specific here.

Reference: [What is this?](http://www.learningjquery.com/2007/08/what-is-this)

File: tests/jsTests/getHelpNowSpec.js (hunk 3 above)
Comment (fatimarafiqui) [reply tests/jsTests/getHelpNowSpec.js:10]: These are mock function calls for supporting jquery's fadeOut and fadeIn. Technically, fadeOut hides and fadeIn shows the element with a delay. I have mocked these function calls to test these functionalities. show and hide functions are functions of jquery. I hope it's clear now.

Reference: [Intro to jasmine](https://jasmine.github.io/2.0/introduction.html) | [hide](http://api.jquery.com/hide/) | [show](http://api.jquery.com/show/)

**Review by MehaKaushik — APPROVED** [review]

File: js/get_help_now.js (hunk 2 above)
Comment (MehaKaushik) [reply js/get_help_now.js:29]: Great !!

**Pull request merged**